src/Linux/Host/Broker/Impl/Services/LinuxRHostProcessService.cs

```diff
@@ -41,15 +45,37 @@ class LinuxRHostProcessService : IRHostProcessService {
             return new UnixProcess(process);
         }
 
+        private string GetUsrLibPath() {
+            const string x64Lib = "/usr/lib/x86_64-linux-gnu";
```

Comment:
Not sure how portable this is. IIRC some distros use `lib64`, and I don't know if there's any reliable standard.

In general, why do we need to try to detect this, instead of using the preset `LD_LIBRARY_PATH` and appending our own value as before? And ditto for `PATH`?